provider/pkg/provider/provider.go

```diff
@@ -63,7 +53,16 @@ import (
 	"k8s.io/client-go/tools/clientcmd"
 	clientapi "k8s.io/client-go/tools/clientcmd/api"
 	k8sopenapi "k8s.io/kubectl/pkg/util/openapi"
+	"net/http"
```

Comment:
nit: We've been using `goimports` to format the k8s provider code. It looks like we may be using different toolchains, so we should pick one and standardize on it.

Reply:
I default to gofmt and assumed make lint would catch it. Sounds good, I will run goimports.